projects/components/src/skeleton/skeleton.component.ts

```diff
@@ -20,6 +20,7 @@ import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/c
     </ng-container>
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.ShadowDom,
```

Comment:
Why do we need this?

Reply:
+1

Reply:
https://angular.io/guide/update-to-version-15#v15-bc-03